### ISSUE TITLE
UvDriver bug in amphp/file 1.0.1

### ISSUE THREAD

**Issue by enumag** [amphp/file#55]
```
[TypeError]                                                                  
Argument 2 passed to Amp\File\StatCache::set() must be of the type array, bool given, called in .../vendor/amphp/file/src/UvDriver.php on line 157 
```

**Comment by enumag** [comment]
I believe it's because you failed to include [this fix](https://github.com/amphp/file/commit/6a506de7add69b6c7b35ff63412a59b0d3b5ab6a) into 1.0.1, effectively breaking support for ext-uv 0.2.

**Comment by trowski** [comment]
Yep, forgot that I had to cherry-pick those from master. Released as 1.0.2.
